openshift/config/crd/kustomization.yaml

```diff
@@ -0,0 +1,41 @@
+# This kustomization.yaml is not intended to be run by itself,
```

Comment:
why do you need this as opposed to upstream files?

Reply:
The reason is patching CRD annotations so we can use our own certificate management instead of cert-manager